examples/token/run.sh

```diff
@@ -0,0 +1,240 @@
+###############################################################################
+"
+leo run transfer_public_to_private aleo17vy26rpdhqx4598y5gp7nvaa9rk7tnvl6ufhvvf4calsrrqdaqyshdsf5z 30u64
+
+# Swap in the private key and address of Bob to .env.
+echo "
+NETWORK=testnet3
+PRIVATE_KEY=APrivateKey1zkpFo72g7N9iFt3JzzeG8CqsS5doAiXyFvNCgk2oHvjRCzF
+" > .env
+
+# Convert 40 private tokens from Bob into 40 public tokens for Alice.
+echo "
+###############################################################################
+########                                                               ########
+########     STEP 6: Convert 40 private tokens from Bob into 40        ########
+########             public tokens for Alice.                          ########
+########                                                               ########
+########           -----------------------------------------           ########
+########           |            PUBLIC BALANCES            |           ########
+########           -----------------------------------------           ########
+########           -----------------------------------------           ########
+########           |        Alice    |         100         |           ########
+########           -----------------------------------------           ########
+########           |        Bob      |          10         |           ########
+########           -----------------------------------------           ########
+########                                                               ########
+########           -----------------------------------------           ########
+########           |            PRIVATE BALANCES           |           ########
+########           -----------------------------------------           ########
+########           -----------------------------------------           ########
+########           |        Alice    |          20         |           ########
+########           -----------------------------------------           ########
+########           |        Bob      |          70         |           ########
+########           -----------------------------------------           ########
+########                                                               ########
+###############################################################################
+"
+leo run transfer_private_to_public "{
+        owner: aleo17vy26rpdhqx4598y5gp7nvaa9rk7tnvl6ufhvvf4calsrrqdaqyshdsf5z.private,
+        amount: 80u64.private,
+        _nonce: 1852830456042139988098466781381363679605019151318121788109768539956661608520group.public
+    }" aleo13ssze66adjjkt795z9u5wpq8h6kn0y2657726h4h3e3wfnez4vqsm3008q 40u64
+
+
+# Swap in the private key and address of the Alice to program.json.
+# This is done to ensure that program.json is the same after every execution of ./run.sh.
+# Swap in the private key and address of the Alice to .env.
```

Comment:
```suggestion
# Swap in the private key and address of Alice to program.json.
# This is done to ensure that program.json is the same after every execution of ./run.sh.
# Swap in the private key and address of Alice to .env.
```

Reply:
I don't see any reference to `program.json` here.

Reply:
Actually these should all be changed to just the "private key"